buildSrc/src/main/kotlin/io/spine/internal/gradle/publish/CheckVersionIncrement.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023, TeamDev. All rights reserved.
+ * Copyright 2024, TeamDev. All rights reserved.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I don't think we need this file at all.

I would be good to review `buildSrc` contents for files like this one, which aren't used.